blackducksoftware/blackduck-alert
# IALERT-3676: add content-security-policy header

Update the security context to remove deprecated methods.  
Add the content security policy header to the security context.

Tests performed:

1. Open Alert UI verify the style of the UI and make sure images loaded.
3. Login to the Alert UI.
4. Create a Black Duck provider configuration.
5. Create a Jira Server configuration.
6. Create a distribution job.
7. In Black Duck generate notifications and make sure the Jira Issue is created and transitioned to Done.
